android/start.md

```diff
@@ -100,19 +109,30 @@ b. To generate the Java classes for these models, click the Gradle Task dropdown
 
 ## Step 3: Add API and Database
 
-a. Run `amplify configure` from the root of your application folder to set up Amplify with your AWS account.
+a. Run `amplify configure` in Terminal from the root of your application folder to set up Amplify with your AWS account.
+
+    - Browser will pop up prompting you to sign in / create new account
+    - Once done, return to terminal and press Enter
+    - Choose a region
+    - Choose a username (can use default)
+    - Browser will popup again to finish creating that user - you can use all default properties.
+    - Leave the page open on the finish step and return to Terminal
+    - Press Enter
+    - It will then ask you to enter the access key ID from the finish page of the browser. Make sure to backspace the default <ACCESS KEY ID> and paste in the key from the browser
```

Comment:
Make sure to backspace the default <ACCESS KEY ID> and copy-paste the key for the IAM user you just created.